services/worker-manager/test/api_test.js

```diff
@@ -1,4 +1,5 @@
 const taskcluster = require('taskcluster-client');
+const slug = require('slugid');
```

Comment:
Could we update whatever test in here asserts that you can fetch a worker via the api to assert that it does _not_ include the field `secret`? As you've written this now it will be fine, I just worry about refactors in the future accidentally exposing this field.